### PR TITLE
Prepare for 1.8.0 release :tada: 

### PR DESCRIPTION
**Pull request by cbgbt**
**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,103 @@
...
+[#2176]: https://github.com/bottlerocket-os/bottlerocket/pull/2176
+[#2178]: https://github.com/bottlerocket-os/bottlerocket/pull/2178
+[#2180]: https://github.com/bottlerocket-os/bottlerocket/pull/2180
+[#2181]: https://github.com/bottlerocket-os/bottlerocket/pull/2181
+[#2183]: https://github.com/bottlerocket-os/bottlerocket/pull/2183
+[#2184]: https://github.com/bottlerocket-os/bottlerocket/pull/2184
+[#2185]: https://github.com/bottlerocket-os/bottlerocket/pull/2185
+[#2187]: https://github.com/bottlerocket-os/bottlerocket/pull/2187
+[#2188]: https://github.com/bottlerocket-os/bottlerocket/pull/2188
+[#2188]: https://github.com/bottlerocket-os/bottlerocket/pull/2188
```
Comment (etungsten) [inline CHANGELOG.md]: Listed twice

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,103 @@
...
+* Restart long-running systemd services on exit ([#2162])
+* Ignore zero blocks on dm-verity root ([#2169])
+* Support adding static DNS mappings to `/etc/hosts` ([#2129])
+* Improve Kubernetes Pod start times by unsetting `configMapAndSecretChangeDetectionStrategy` in Kubelet config ([#2166])
+* Add support for non-`eth0` default interfaces ([#2144])
+* Build smartpqi driver for Microchip Smart Storage devices into 5.10 kernel ([#2184])
+* Add support for Broadcom ethernet cards in 5.10 kernel ([#2143])
+* Add support for MegaRAID SAS in 5.10 kernel ([#2133])
+* Add new `provider-id` Kubelet setting ([#2192])
+* Add new `podPidsLimit` Kubelet setting ([#2138])
```
Comment (etungsten) [inline CHANGELOG.md]: nit:
```suggestion
* Add new setting for configuring kubelet's `providerID` configuration ([#2192])
* Add new setting for configuring kubelet's `podPidsLimit` configuration ([#2138])
```

**Review by etungsten — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,103 @@
+# v1.8.0 (2022-06-08)
+
+## OS Changes
+* Add support for Kubernetes 1.23 variants ([#2188])
+* Add aws-ecs-1-nvidia variant with Nvidia driver support ([#2128], [#2100], [#2098], [#2167], [#2097], [#2090], [#2099])
+* Update admin and control containers ([#2191])
+* Update to containerd 1.6.x ([#2158])
+* Restart container runtimes when certificates store changes ([#2076])
+* Add support for ECS ImagePullBehavior and WarmPoolsSupport ([#2063], thanks, @mello7tre!)
+* Add support for providing Kernel parameters via Boot Configuration ([#1980])
```
Comment (bcressey) [inline CHANGELOG.md]: link is not working

**Review by zmrow — APPROVED** [review]
It's a big one! :partying_face: :tada:

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,103 @@
+# v1.8.0 (2022-06-08)
+
+## OS Changes
+* Add support for Kubernetes 1.23 variants ([#2188])
+* Add aws-ecs-1-nvidia variant with Nvidia driver support ([#2128], [#2100], [#2098], [#2167], [#2097], [#2090], [#2099])
```
Comment (bcressey) [inline CHANGELOG.md]: Do we want to consider adding sub-sections, like `General`, `ECS`, `Kubernetes`? Different readers will find different features exciting, and right now they're all mixed together.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,103 @@
...
+* Update admin and control containers ([#2191])
+* Update to containerd 1.6.x ([#2158])
+* Restart container runtimes when certificates store changes ([#2076])
+* Add support for ECS ImagePullBehavior and WarmPoolsSupport ([#2063], thanks, @mello7tre!)
+* Add support for providing Kernel parameters via Boot Configuration ([#1980])
+* Restart long-running systemd services on exit ([#2162])
+* Ignore zero blocks on dm-verity root ([#2169])
+* Support adding static DNS mappings to `/etc/hosts` ([#2129])
```
Comment (bcressey) [inline CHANGELOG.md]: Unless we're varying the language for stylistic reasons:
```suggestion
* Add support for static DNS mappings in `/etc/hosts` ([#2129])
```

File: CHANGELOG.md (hunk 3 above)
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Add support for providing kernel parameters via Boot Configuration ([#1980])
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,103 @@
...
+* Improve Kubernetes Pod start times by unsetting `configMapAndSecretChangeDetectionStrategy` in Kubelet config ([#2166])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Improve Kubernetes pod start times by unsetting `configMapAndSecretChangeDetectionStrategy` in kubelet config ([#2166])
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,103 @@
...
+* Add support for non-`eth0` default interfaces ([#2144])
+* Build smartpqi driver for Microchip Smart Storage devices into 5.10 kernel ([#2184])
+* Add support for Broadcom ethernet cards in 5.10 kernel ([#2143])
+* Add support for MegaRAID SAS in 5.10 kernel ([#2133])
+* Add new `provider-id` Kubelet setting ([#2192])
+* Add new `podPidsLimit` Kubelet setting ([#2138])
+* Fix `settings.kubernetes.cluster-dns-ip` to allow list of IP addresses ([#2176])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Allow a list of IP addresses in `settings.kubernetes.cluster-dns-ip` ([#2176])
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,103 @@
...
+* Deny unknown fields in network configurations ([#2142])
+* Enable network configuration generation via `netdog` ([#2066])
+* Require `enabled` key in DHCP configuration ([#2146])
```
Comment (bcressey) [inline CHANGELOG.md]: Likewise, this seems too trivial:
```suggestion
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,103 @@
...
+* Support adding static DNS mappings to `/etc/hosts` ([#2129])
+* Improve Kubernetes Pod start times by unsetting `configMapAndSecretChangeDetectionStrategy` in Kubelet config ([#2166])
+* Add support for non-`eth0` default interfaces ([#2144])
+* Build smartpqi driver for Microchip Smart Storage devices into 5.10 kernel ([#2184])
+* Add support for Broadcom ethernet cards in 5.10 kernel ([#2143])
+* Add support for MegaRAID SAS in 5.10 kernel ([#2133])
+* Add new `provider-id` Kubelet setting ([#2192])
+* Add new `podPidsLimit` Kubelet setting ([#2138])
+* Fix `settings.kubernetes.cluster-dns-ip` to allow list of IP addresses ([#2176])
+* Deny unknown fields in network configurations ([#2142])
```
Comment (bcressey) [inline CHANGELOG.md]: This seems too minor to feature in the changelog:
```suggestion
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,103 @@
...
+* Add support for ECS ImagePullBehavior and WarmPoolsSupport ([#2063], thanks, @mello7tre!)
+* Add support for providing Kernel parameters via Boot Configuration ([#1980])
+* Restart long-running systemd services on exit ([#2162])
+* Ignore zero blocks on dm-verity root ([#2169])
+* Support adding static DNS mappings to `/etc/hosts` ([#2129])
+* Improve Kubernetes Pod start times by unsetting `configMapAndSecretChangeDetectionStrategy` in Kubelet config ([#2166])
+* Add support for non-`eth0` default interfaces ([#2144])
+* Build smartpqi driver for Microchip Smart Storage devices into 5.10 kernel ([#2184])
+* Add support for Broadcom ethernet cards in 5.10 kernel ([#2143])
+* Add support for MegaRAID SAS in 5.10 kernel ([#2133])
```
Comment (bcressey) [inline CHANGELOG.md:33]: Maybe a "Hardware" sub-section for these? Or condense into a single line item?

**Comment by cbgbt** [comment]
Force push to address feedback from @etungsten and @bcressey.

**Review by etungsten — APPROVED** [review]

**Comment by cbgbt** [comment]
Force push again to fix a comment that I missed.

**Review by jpculp — APPROVED** [review]

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,3 +1,108 @@
...
+* Update admin and control containers ([#2191])
+* Update to containerd 1.6.x ([#2158])
+* Restart container runtimes when certificates store changes ([#2076])
+* Add support for providing kernel parameters via Boot Configuration ([#1980])
+* Restart long-running systemd services on exit ([#2162])
+* Ignore zero blocks on dm-verity root ([#2169])
+* Add support for static DNS mappings in `/etc/hosts` ([#2129])
+* Enable network configuration generation via `netdog` ([#2066])
+* Add support for non-`eth0` default interfaces ([#2144])
+* Add c7g instance type support to AWS variants ([#2107], thanks, @lizthegrey!)
```
Comment (bcressey) [inline CHANGELOG.md]: This is only relevant for Kubernetes, and it's closer to
```suggestion
* Add c7g instance data for max pods calculation in AWS variants ([#2107], thanks, @lizthegrey!)
```

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,3 +1,108 @@
...
+* Update to IMDS schema `2021-07-15` ([#2190])
+
+### Kubernetes
+* Add support for Kubernetes 1.23 variants ([#2188])
+* Improve Kubernetes pod start times by unsetting `configMapAndSecretChangeDetectionStrategy` in kubelet config ([#2166])
+* Add new setting for configuring kubelet's `provider-id` configuration ([#2192])
+* Add new setting for configuration kubelet's `podPidsLimit` configuration ([#2138])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Add new setting for configuring kubelet's `podPidsLimit` configuration ([#2138])
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,3 +1,108 @@
...
+* Allow a list of IP addresses in `settings.kubernetes.cluster-dns-ip` ([#2176])
+* Set the default for `settings.kubernetes.cloud-provider` on metal variants to be an empty string ([#2188])
```
Comment (bcressey) [inline CHANGELOG.md]:
```suggestion
* Set the default for `settings.kubernetes.cloud-provider` on metal variants to an empty string ([#2188])
```

File: CHANGELOG.md (hunk 11 above)
Comment (cbgbt) [reply CHANGELOG.md]: Serves me right for not looking at this more closely! I'll fix it.

**Comment by cbgbt** [comment]
Fixed issues pointed out by @bcressey.

**Review by bcressey — APPROVED** [review]

**Pull request merged**